domains/lightarea.is-cool.dev.json

```diff
@@ -7,8 +7,7 @@
     },
 
     "record": {
-       "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
-        "TXT": ["v=spf1 include:spf.improvmx.com ~all"]
+        "CNAME": "lighthostingfree.github.io"
```

Comment:
CNAME returns 404 page.

Reply:
> CNAME returns 404 page.

Bro website is at lighthostingfree.github.io/LightArea